File: DRRR.Server/Dtos/PaginationDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DRRR.Server.Dtos
{
    public class PaginationDto
    {
        public int CurrentPage { get; set; }

        public int TotalPages { get; set; }

        public int TotalItems { get; set; }
    }
}

File: DRRR.Client/src/app/chat-rooms/dtos/pagination.dto.ts
/**
 * 分页信息DTO
 */
export interface PaginationDto {
  /**
   * 当前页码
   */
  currentPage: number;

  /**
   * 总页数
   */
  totalPages: number;

  /**
   * 总项目数
   */
  totalItems: number;
}
